source/_cookbook/turn_on_light_for_10_minutes_when_motion_detected.markdown

```diff
@@ -18,6 +18,7 @@ automation:
   action:
     service: light.turn_on
     entity_id: light.kitchen_light
+  mode: single
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

As this is an optional parameter, it should not be part of these examples.